app/helpers/assistance_box_helper.rb

```diff
@@ -6,4 +6,8 @@ def assistance_box(assignment)
       </div>}.html_safe
     end
   end
+
+  def assistance_rules_passed?(assignment)
+    Mumukit::Assistant. parse(assignment.exercise.assistance_rules).rules.any? { |rule| rule.matches? assignment }
```

Comment:
Let's please move this logic to mumukit assistant if possible.
Also we could rename it to `should_display_assistance_box?`

Reply:
Also please remember there is already an `assistant` property with the rules parsed: 

https://github.com/mumuki/mumuki-domain/blob/dc34611ace2ece06a66590d0ae017f083b84ff27/app/models/concerns/assistable.rb#L10

Thus, we should write this code as `assistant.rules.any?  ...`

Reply:
Sorry, maybe my last comment wasn't clear enough. There's no necessary for this code neither, cause using `tips.present?` in assignment is sufficient.